### DISCUSSION TITLE
Add git clone support

### DISCUSSION THREAD

**Discussion by waelchateur** [tyron12233/CodeAssist#172] [Ideas]
add a way to use git clone

already tryed to compile a simple rerpository that i copied in data projects folder but it fail and it seems that i need to follow directory structure as the simple project that the app give so it can be compiled correctly

and one more thing the app seems to be not able to compile projects with support library 🥴

**Comment by tyron12233** [comment]
are you on the latest version?

**Comment by waelchateur** [comment]
@tyron12233 yes

**Comment by hosnifraj0** [comment]
> add a way to use git clone
> 
> already tryed to compile a simple rerpository that i copied in data projects folder but it fail and it seems that i need to follow directory structure as the simple project that the app give so it can be compiled correctly
> 
> and one more thing the app seems to be not able to compile projects with support library 🥴

Can you send me the repo link ?

**Comment by Irineu333** [comment]
I couldn't clone too, I needed to download the zip
